android/build.gradle

```diff
@@ -9,7 +9,7 @@ buildscript {
     dbFlowVersion = '4.2.4'
     buildToolsVersion = '28.0.0'
     supportLibVersion = '28.0.0'
-    kotlinVersion = '1.3.50'
+    kotlinVersion = '1.3.61'
```

Comment:
Should be done in separate PR IMO 🤔 And I wish it would be alongside some other libs/deps update 😉